scripts/functional-tests.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 # SPDX-License-Identifier: Apache-2.0
+#
+# If you are running locally and calling the script multiple times you may want to run like this:
+#
+# CLEANUP_TEST_DIR=1 TEST_DIR=/tmp/foo ./scripts/functional-tests.sh
```

Comment:
When would I want to *not* set `CLEANUP_TEST_DIR=1`? The directory name is uniquely generated on each script run. Leaving the downloaded files inside `$TEST_DIR` won't speed up the next invocation anyway. What am I missing? I question the need for the variable.